### DISCUSSION TITLE
Notifications of deployment failure to any email address?

### DISCUSSION THREAD

**Discussion by jeremy303** [vercel/community#1301] [Idea]
### Summary

**What:**  We need to have notifications of deployment errors emailed to a mailing list, rather than just to the email address of the team member account associated with the automated deployment.  I was hoping the new notification features in beta might support this, but it seems it not.

**Why:**  Our deployments are triggered by a CMS, whenever new content is published--  not just by a developer pushing a software update.  Currently there is no way for content authors (and there are over a dozen) to know if the publish failed due to an error with the Vercel deployment.  To work around this, we'd like to add all content authors to a mailing list that could receive email notifications of deployment errors.

Any ideas (aside from setting up a forwarding rule in my email client)?   If it's not hiding somewhere already, support for notifications to any email address would be incredibly valuable for use cases such as this.

**Comment by amyegan** [comment]
This is a great idea @HolySamosa! I'm moving it into the "Ideas" category and sharing it with the team
